portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,18 @@ function sayHello(){
   });
 }
 
+//retrieves json comments from server 
+function loadComments(){
+    fetch('/data').then(response => response.json()).then((comments) => {
+      const allCommentsList = document.getElementById('comments-container');
+      comments.forEach((sentComment) => {
+          let allComments = document.createElement('p');
```

Comment:
It seems to me like this would be more of a `let comment = document.createElement('p');` just seeing as you have a `comments.forEach()`, but I could be wrong.

Reply:
yep i changed it to singleComment